src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -2832,6 +2833,19 @@ void AuraEffect::HandleAuraMounted(AuraApplication const* aurApp, uint8 mode, bo
                 creatureEntry = 15665;
         }
 
+        // Festive Brewfest Mount
+        if (!GetBase()->HasEffectType(SPELL_AURA_MOD_INCREASE_MOUNTED_FLIGHT_SPEED) && target->HasAura(66052))
+        {
+            if (caster->GetSpeedRate(MOVE_RUN) >= 2.0f)
+            {
+                creatureEntry = 27707;
```

Comment:
we should make an enum with all constants of this and remove these magic numbers
like
```c++
enum brewfestEntries = {
  GREAT_BREWFEST_KODO = 27707,
  npc_name_2 = 30507,
  FESTIVE_BREWFEST_MOUNT = 66052,
  ... and so on ...
}
```